components/LoginDialog.js

```diff
@@ -88,7 +106,7 @@ class LoginDialog extends Component {
           <div className="primary">
             <a className="button link" href="/forgot-password">Forgot password?</a>
             <button
-              disabled={!this.state.email || !this.state.password || this.props.loggingIn}
```

Comment:
Is the removal if `this.state.` intentional?

Reply:
yes because email and password are destructured from state on L#56-60